pandas/tseries/holiday.py

```diff
@@ -157,15 +157,34 @@ class from pandas.tseries.offsets
         --------
         >>> from pandas.tseries.holiday import Holiday, nearest_workday
         >>> from dateutil.relativedelta import MO
-        >>> USMemorialDay = Holiday('Memorial Day', month=5, day=31,
-                                    offset=pd.DateOffset(weekday=MO(-1)))
-        >>> USLaborDay = Holiday('Labor Day', month=9, day=1,
-                                offset=pd.DateOffset(weekday=MO(1)))
-        >>> July3rd = Holiday('July 3rd', month=7, day=3,)
-        >>> NewYears = Holiday('New Years Day', month=1,  day=1,
-                               observance=nearest_workday),
-        >>> July3rd = Holiday('July 3rd', month=7, day=3,
-                              days_of_week=(0, 1, 2, 3))
+
+        >>> USMemorialDay = Holiday(
```

Comment:
does a code check complain about this without this change?  this makes it harder to copy/paste an example

Reply:
Yes, flake8 complains that this line's length is larger than 88 characters.